src/ngraph/runtime/gpu/gpu_kernel_emitters.hpp

```diff
@@ -26,6 +28,131 @@ namespace ngraph
         {
             namespace kernel
             {
+                void emit_prologue(codegen::CodeWriter& writer, const Node* node)
```

Comment:
These two functions already exist in the code_writer, use those.

Reply:
Switched to codegen::CodeWriter::block_begin/end. It appears that CodeWriter::block_begin/end is not frequently used for emitters in the cpu ([e.g.](https://github.com/NervanaSystems/ngraph/blob/cs%2Fgpu_sum/src/ngraph/runtime/cpu/cpu_emitter.cpp#L135)) and gpu transformers because a block comment is often desired. To this end I added prefix/suffix default parameters to CodeWriter::block_begin/end so that this functionality is captured.

Reply:
Block comment is on the chopping block. They will be removed.